server/admin/member.js

```diff
@@ -108,14 +108,16 @@ function createReapHandler(ringpop) {
             return;
         }
 
+        var reaped = 0;
         ringpop.membership.members.forEach(function(member) {
             if (member.status === Member.Status.faulty) {
+                reaped++;
                 ringpop.membership.makeTombstone(member.address, member.incarnationNumber);
             }
         });
 
         process.nextTick(function() {
-            callback(null, null, JSON.stringify({'status': 'ok'}));
```

Comment:
I think we also need `'status': 'ok'` in here, to signal that the operation succeeded. The number of reaped nodes is only extra information.